### PR TITLE
VY 122 Add employee tabbed form

### PR DESCRIPTION
**Pull request by lf-romitamgai**
@lf-kiranpariyar @joshianuj please review the code

### CODE REVIEW

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: frontend/src/js/actions/employeeActions.js [hunk 1]
```diff
@@ -59,6 +64,12 @@ let employeeActions = {
       key: key,
       value: value
     }
+  }, selectItem: function (entity, data) {
```
Comment (lf-kiranpariyar) [inline frontend/src/js/actions/employeeActions.js]: convert this line to ES6.

File: frontend/src/js/components/employee/employee-form/EmployeeForm.js [hunk 2]
```diff
@@ -278,363 +218,48 @@ class EmployeeForm extends React.Component {
...
+                  <Tabs
+                    onSelect={this.handleSelect}
+                    selectedIndex={this.state.selectedTab}
+                  >
+                    <TabList>
+                      <Tab>General Information</Tab>
+                      <Tab disabled={!this.state.isValidInfo || this.state.infoChange}>HR Information</Tab>
+                    </TabList>
+                    <TabPanel>
+                      <GeneralInfo
```
Comment (pratishshr) [inline frontend/src/js/components/employee/employee-form/EmployeeForm.js]: use object destructuring

File: frontend/src/js/components/employee/employee-form/tabbed-components/GeneralInfo.js [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    }
+  }
+
+  getRequiredFields() {
+    let requiredFields = {
+      firstName: this.refs.firstName.value,
+      lastName: this.refs.lastName.value,
+      gender: this.refs.gender.value,
+      primaryEmail: this.refs.primaryEmail.value,
+      maritalStatus: this.refs.maritalStatus.value == "0" ? '' : this.refs.maritalStatus.value
```
Comment (pratishshr) [inline frontend/src/js/components/employee/employee-form/tabbed-components/GeneralInfo.js]: use direct (this.refs.maritalStatus.value)?

**Pull request merged**